product_docs/docs/pgd/5/limitations.mdx

```diff
@@ -108,6 +108,27 @@ Be sure to disable transaction streaming when planning to use
 CAMO. You can configure this option globally or in the PGD node group. See
 [Transaction streaming configuration](../transaction-streaming#configuration).
 
+-   Not all DDL can run when CAMO is used. The only supported DDL operations are:
```

Comment:
Oh, I think we could be more specific here: CAMO and commit scopes gets automatically disabled for DDL that it does not support. You'll get for example:

```
WARNING:  0A000: commit scopes automatically disabled for ALTER TABLE
DETAIL:  This transaction's commit scope will be local only.
```

And yes, this is stupid. But no, we did not invent it. It was added upon specific (important) customer's request. :-) 

Same applies for Eager.

Reply:
Done with a small Eager section reffing the GC entry to avoid repeating the same info three times.